support-files/mariadb.service

```diff
@@ -0,0 +1,104 @@
+#
+#  /etc/systemd/system/mariadb.service
+#
+
+[Unit]
+Description=MariaDB database server
+After=network.target
+After=syslog.target
```

Comment:
Really? I thought the whole idea of socket activation is to remove the need
for explictly specified service dependencies.